src/models/BookModel.ts

```diff
@@ -4,6 +4,7 @@ import { MediaType } from '../utils/MediaType';
 
 export class BookModel extends MediaTypeModel {
 	author: string;
+	plot:string;
```

Comment:
small formatting mistake, running `npm run format` should fix this

Reply:
Ah good to know, I ran it and it also fixed a line end in MovieModel.ts